admin/config-ui/class-configuration-structure.php

```diff
@@ -49,8 +49,12 @@ public function initialize() {
 		$this->add_step( 'postTypeVisibility', __( 'Post type visibility', 'wordpress-seo' ), $fields );
 
 		$this->add_step( 'multipleAuthors', __( 'Multiple authors', 'wordpress-seo' ), array( 'multipleAuthors' ) );
-		$this->add_step( 'connectGoogleSearchConsole', __( 'Google Search Console', 'wordpress-seo' ), array( 'connectGoogleSearchConsole' ) );
+		$this->add_step( 'connectGoogleSearchConsole', __( 'Google Search Console', 'wordpress-seo' ), array(
```

Comment:
We might consider `sprintf` the value `Google` because that value shouldn't be translatable.

Reply:
This isn't done anywhere else in the plugin and doing so partially seems confusing to both translators and developers.